scripts/generateAssetData/ethereum/uniswapV2Pools.ts

```diff
@@ -105,7 +105,7 @@ export const getUniswapV2Pools = async () => {
         name,
         precision: Number(appTokenData.decimals),
         icons,
-        icon: 'https://assets.coincap.io/assets/icons/256/uni.png',
+        icon: 'https://assets.coingecko.com/coins/images/12504/standard/uni.jpg?1696512319',
```

Comment:
```suggestion
        icon: 'https://coin-images.coingecko.com/coins/images/12504/large/uni.jpg?1696512319',
```

Reply:
Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)